### ISSUE TITLE
Add more installation detail

### ISSUE THREAD

**Issue by mgdenno** [RTIInternational/teehr#306]
The installation section in getting started should have a bit more explanation.  A discussion on virtual environments, etc.  I the user just copy-pasts the code i think it should give them a working installation.  I'd be inclined to stick with he basics.

```
mkdir teehr-getting-started
cd teehr-getting-started
python3 -m venv .venv
source .venv/bin/activate
pip install 'teehr @ git+https://github.com/RTIInternational/teehr@v0.4.0'
```

I think we could remove the poetry part and just keep it simple.  Users can add complexity if the know how.

**Comment by mgdenno** [comment]
This is just an idea, might be worth looking at other repos to see how they do it.

**Comment by mgdenno** [comment]
@samlamont I will work on this while I am in here making other changes.

**Comment by samlamont** [comment]
@mgdenno Sounds good, I was thinking we could potentially make the whole page a notebook, so we could execute the example at the bottom (and add more detail, e.g., plots?). Not sure if it's worth the effort though since it breaks the links with other pages.  I can take a quick look to see how feasible it is

**Comment by mgdenno** [comment]
Yeah, there seems to be a balance here as the notebooks are not quite as robust as the actual docs pages.

**Comment by mgdenno** [comment]
This was addressed in v0.4.0.
